Header header

float32 Tyaw
float32 Dyaw
float32 Tpitch
float32 Dpitch

geometry_msgs/Vector3 magnetic
float64[9] magnetic_covariance # Row major about x, y, z axes

geometry_msgs/Vector3 gravity
float64[9] gravity_covariance # Row major x, y z 

